src/modules/75-cf/hooks/useEnvironmentSelectV2.tsx

```diff
@@ -104,10 +105,18 @@ export const useEnvironmentSelectV2 = ({
   }, [preferredEnvironment, selectedEnvironment?.value])
 
   useEffect(() => {
+    if (allEnvironments) {
+      const allEnvValue = {
+        label: getString('common.allEnvironments'),
+        value: getString('common.allEnvironments')
+      }
+      setSelectedEnvironment(allEnvValue)
+      refetchProjectFlags()
+      return
+    }
```

Comment:
Super minor, but we could do with a line break here to break up the wall of code 👍🏻
```suggestion
    }

```

Reply:
cool, thanks have updated